desktop/app.js

```diff
@@ -153,23 +154,38 @@ function createMenuTemplate( settings ) {
 	var fileMenu = {
 		label: '&File',
 		submenu: [ {
-				label: '&New Note',
-				accelerator: 'CommandOrControl+N',
-				click( item, focusedWindow ) {
-					if ( focusedWindow ) {
-						focusedWindow.webContents.send( 'appCommand', { action: 'newNote' } );
-					}
+			label: '&New Note',
+			accelerator: 'CommandOrControl+N',
+			click( item, focusedWindow ) {
+				if ( focusedWindow ) {
+					focusedWindow.webContents.send( 'appCommand', { action: 'newNote' } );
 				}
-			}, {
-				type: 'separator'
-			}, {
-				label: '&Print',
-				accelerator: 'CommandOrControl+P',
-				click( item, focusedWindow ) {
-					if ( focusedWindow ) {
-						focusedWindow.webContents.send( 'appCommand', { action: 'setShouldPrintNote' } );
-					}
+			}
+		}, {
+			type: 'separator'
+		}, {
+			label: '&Export',
```

Comment:
I did start with `Export All Notes` but it seemed very odd in the menu against the other shorter entries. I'll have a look at `Export Notes`